src/utils/createPptx/pptxConfiguration/slideElements.ts

```diff
@@ -67,8 +67,8 @@ export const imageStyling: PptxGenJS.ImageProps = {
 
 export const citeAsStyling: PptxGenJS.TextPropsOptions = {
   x: '70%',
```

Comment:
The PR gets some build errors, so I can't try this out, but doesn't this look off? This places the citeAs to start at 70% of the slide's width, doesn't it?

Reply:
Yes it puts it down in the right corner.
Should we keep it as a long string at the bottom instead?